packages/config/src/Config.ts

```diff
@@ -0,0 +1,89 @@
+import * as enums from './enums';
+import { Autocomplete } from './Autocomplete'
+import { Recommendation } from './Recommendation'
+import { Search } from './Search'
+import { Content } from './Content'
+import { Method } from '@findify/sdk/types/request'
+
+export type Config = {
+  /** Merchant API key */
+  key: string
+
+  /** Merchant ID, required to connect Chrome DevTools */
+  merchant_id: number
+
+  /** Store status */
+  status: enums.StoreStatus
+
+  /** Store platform. Generic by default */
+  platform: {
```

Comment:
@Bitaru mb change it to simply: `platform: keyof typeof enums.Platform` ?

Reply:
+